File: C++/Hardware.cpp
#include <string>
#include "Product.cpp"

using namespace std;

class Hardware : public Product
{
private:
	//deklarasi
	string model;
	string brand;

public:
	//Constructor 
	Hardware(string model, string brand)
	{
		this->model = model;
		this->brand = brand;
	};

  	//Constructor 
	Hardware(){

	};

	// get set
	void setModel(string model)
	{
		this->model = model;
	};

	string getModel()
	{
		return this->model;
	};

	void setBrand(string brand)
	{
		this->brand = brand;
	};

	string getBrand()
	{
		return this->brand;
	};
};

File: Python/Hardware.py
from Product import Product

class Hardware(Product):
    #atribut 
    __brand = ""
    __model = ""

    #constructor
    def __init__(self):
        self.__brand = ""
        self.__model = ""
    
    #set get
    def setBrand(self, brand):
        self.__brand = brand
    def getBrand(self):
        return self.__brand
    
    def setModel(self, model):
        self.__model = model
    def getModel(self):
        return self.__model